Cairo Desktop/CairoDesktop.Localization/Language.cs

```diff
@@ -166,7 +166,7 @@ static class Language
             { "sSettings_Desktop_LabelPositionBottom", "Bottom" },
             { "sSettings_Desktop_DynamicDesktop", "Dynamic Desktop" },
             { "sSettings_Desktop_EnableDynamicDesktop", "Use Dynamic Desktop" },
-            { "sSettings_Desktop_EnableDesktopOverlayHotKey", "Enable desktop overlay hotkey:" },
+            { "sSettings_Desktop_EnableDesktopOverlayHotKey", "Enable active desktop overlay hotkey:" },
```

Comment:
Should this say dynamic instead of active?

Reply:
Yes of course.
Can you correct this please?
Thanks